src/quo2/foundations/colors.cljs

```diff
@@ -246,7 +246,8 @@
             base-color    (get-in colors-map
                                   [(if (= color-keyword :yinyang)
                                      (if (theme/dark?) :yang :yin)
-                                     (keyword color)) suffix])]
+                                     (keyword color)) suffix]
+                                  color)]
```

Comment:
hi @codemaster115, Thank you very much for your work.
Why we are returning color keyword here?

Reply:
@Parveshdhull In case of color value is literal like hex, it will return `nil` value instead of the origin.

Reply:
If we already have hex value, why we are using this function. Shouldn't this function is used only when we have color keyword and want color hex?

Reply:
To use predefined colors in the Status theme together, it's appropriate to use this function.